src/Host.h

```diff
@@ -577,6 +577,7 @@ class Host : public QObject
     double mLineSize;
     double mRoomSize;
     bool mShowInfo;
```

Comment:
TBH You do not need to keep this member any more:
* when reading the profile if you do **not** see the new setting that holds the value of `"mMapInfoContributors"` then if you read the `"mShowInfo"` you set `mMapInfoContributors` to be in the "none" mode if it is "no" and "full" if it is "yes". If you do see the new setting than you use it as you plan.
* when saving the profile if `mMapInfoContributors` is in the "none" mode then you save the `"mShowInfo"` as "no", you **do not write that setting if it anything else**. In either case you save the `mMapInfoContributors` as planned.

This will mean that during the change over period old Mudlets show either none or full if they are loaded with a save from the new version depending on whether the new version saved with nothing or something showing. New versions will get the new setting if loaded from a version previous saved from a new version and none or full if loaded from a version saved by an old one.

Reply:
You are totally right :)

Reply:
Should be working like you've described..